src/Compilers/Core/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    // PROTOTYPE(nullable-api): Document
+    public enum NullableAnnotation : byte
+    {
+        NotApplicable = 0,
+        Disabled,      // No information. Think oblivious.
```

Comment:
>Disabled [](start = 8, length = 8)

`Disabled` seems misleading because it seems to imply that annotations were ignored rather than the more likely scenario that annotations were never included. I think this should be `None` or `Unset`.

Reply:
Noted. These APIs will need to go through a formal design review, so I'll bring up your concerns there. This is what the current plan of record has, so it's what I'm going with for now.

---
In reply to: [266024732](https://github.com/dotnet/roslyn/pull/34017#discussion_r266024732) [](ancestors = 266024732)